cmd/earthly/subcmd/satellite_cmds.go

```diff
@@ -987,8 +988,30 @@ func (a *Satellite) actionUpdate(cliCtx *cli.Context) error {
 		return errors.Errorf("not a valid platform: %q", platform)
 	}
 
+	if sat.State != cloud.SatelliteStatusSleep {
+		a.cli.Console().Printf("")
+		a.cli.Console().Printf("The satellite must be asleep to start the update.")
+		a.cli.Console().Printf("Putting the satellite to sleep will interrupt any running builds.")
+		a.cli.Console().Printf("")
+		answer, err := common.PromptInput(cliCtx.Context, "Would you like to put it to sleep now? [y/N]: ")
```

Comment:
can we have a `--force` option to override this

Reply:
I would go even further and never make anything interactive, because it makes scripting more difficult. It's probably best to just never have this kind of option and it's always forced. It's not like someone will be like "oh I didn't know that it was sleeping... maybe I won't bother it then" :-)

Reply:
we can alternatively call https://github.com/earthly/earthly/blob/761740f72931261bf6f7a10168d006e398388057/util/termutil/is_tty.go#L6

Reply:
I think Alex's original suggestion to have a `--force` option makes sense. And perhaps it just fails without `--force`, if it's not already asleep. The failure message should perhaps include the suggestion to use `--force`.

BTW, is there a good way to tell whether a build is actually ongoing, and only fail in that case (but not if the satellite is awake and idle)? This might need some sort of support end-to-end to make it atomic perhaps.

Reply:
We can detect if there are no builds ongoing, but to make it atomic, we'd need backend support (as you mentioned). I think the backend support could be bit for work than it's worth (e.g. different API that supports streaming, or some way to lock the satellite momentarily blocking new builds).

I think I would vote for the error message just telling the user to re-run with `--force` , but personally I do think the interactive mode will be more user-friendly in most cases (with `--force` still being script-friendly)

Reply:
Makes sense. BTW we made the decision to avoid interactive for many other commands. We only kept it for things that are never going to be used for scripting for sure (like account registration). I guess in this specific case, it's a consistency argument more than anything.

Reply:
Fair enough - I wasn't sure if this would be a common command to script, but I see the value in consistency. 

Shall we proceed with showing an error and prompting to run with `--force`  if the satellite is awake instead?

Reply:
SGTM!